app/controllers/users_controller.rb

```diff
@@ -0,0 +1,25 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new
+    @user.email = user_params[:email]
```

Comment:
Do we need to be this explicit for these attributes?  Would it be better if we pass `user_params` directly into `User.new`